events/wc_frostwolf_events.txt

```diff
@@ -338,43 +380,114 @@ province_event = {
 		d_frostwolf = {
 			holder_scope = {
 				if = {
-					limit = { frostwolf_exodus_d_frostwolf_trigger = yes }
-					narrative_event = { id = WCFW.16 }
+					limit = { frostwolf_leader_exists = yes }
+					narrative_event = { id = WCFW.21 }
 				}
 				else = {
-					character_event = { id = WCFW.200 }
+					73 = { province_event = { id = WCFW.600 } }
 				}
 			}
 		}
 	}
 }
+
 # Delayed event
 narrative_event = {
-	id = WCFW.16
-	title = EVTTITLE_WCFW_16
-	desc = EVTDESC_WCFW.16 #We lose the battle!
-	picture = GFX_evt_dwarf_against_orc_win
+	id = WCFW.21
+	title = EVTTITLE_WCFW_21
+	desc = EVTDESC_WCFW_21 # After scouting the coastline, we found a small, poorly protected port in which many merchant ships are moored
+	picture = GFX_evt_busy_trading_dock_republic
 	border = GFX_event_narrative_frame_war
 
 	is_triggered_only = yes
 
 	option = {
-		name = EVTOPTA_WCFW.16
-		prestige = -20
-		add_trait_silently_wounded_effect = yes
-
+		name = EVTOPTA_WCFW_21 # Go ahead!
+		custom_tooltip = { text = EVTOPTA_WCFW_21_tooltip } # We steal the ships.
+		
 		73 = {
 			province_event = {
-				id = WCFW.114 #Western
-				days = 21
+				id = WCFW.22
```

Comment:
`WCFW.22` isn't `province_event`, it's `narrative_event` so the event chain stops here.

Reply:
Should be resolved, delayert event `WCFW.122` was missing for `WCFW.22`.